js/chrome-tabs.js

```diff
@@ -250,6 +263,25 @@
       }
     }
   }
+  
+  var extend = function() {
+    var extended = {};
+
+    var merge = function(obj) {
+      for (var prop in obj) {
+        if (Object.prototype.hasOwnProperty.call(obj, prop)) {
+          extended[prop] = obj[prop];
+        }
+      }
+    };
+
+    for (var i = 0; i < arguments.length; i++) {
+      var obj = arguments[i];
+      merge(obj);
+    }
+
+    return extended;
+  };
```

Comment:
This can be removed if we replace `extend`’s use with `Object.assign` as mentioned above.

Additionally, were we to accept this, it would need to be written in ES6 syntax to match the rest of this file. So things like `const extend = () => ...` as opposed to `var extend = function() ...`.